examples/e2e/README.md

```diff
@@ -0,0 +1,91 @@
+# Introduction
```

Comment:
Should we create a section in the main readme that points to these examples?

Reply:
Done